packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -1078,8 +1082,8 @@ export default class PciInstancesAddController {
   }
 
   displayNetwork(mode, type) {
-    if (type === this.LOCAL_ZONE_REGION) return mode !== 'public_mode';
-    if (type !== this.LOCAL_ZONE_REGION) return mode !== 'local_private_mode';
+    if (type === this.LOCAL_ZONE_REGION) return mode !== MODE_PUBLIC;
+    if (type !== this.LOCAL_ZONE_REGION) return mode !== MODE_LOCAL_PRIVATE;
     return true;
```

Comment:
return true is never call, you can simplify it
```suggestion
    return  (type === this.LOCAL_ZONE_REGION) ?  mode !== MODE_PUBLIC : !this.isLocalPrivateMode();
```

Reply:
ok simplified all